{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NC_019713.1 converted\n",
      "KJ572844.2 converted\n",
      "KP280062.1 converted\n",
      "NC_019722.1 converted\n",
      "KM612265.1 converted\n",
      "KM612263.1 converted\n",
      "KM612262.1 converted\n",
      "KM612260.1 converted\n",
      "NC_027393.1 converted\n",
      "KM612261.1 converted\n",
      "NC_015159.1 converted\n",
      "NC_027397.1 converted\n",
      "NC_013651.1 converted\n",
      "KP280063.1 converted\n",
      "NC_015157.1 converted\n",
      "NC_015158.1 converted\n",
      "NC_016567.1 converted\n",
      "NC_005891.1 converted\n",
      "KP671755.1 converted\n",
      "KJ545483.2 converted\n",
      "HQ317393.2 converted\n",
      "NC_005083.2 converted\n",
      "NC_021073.1 converted\n",
      "NC_020488.1 converted\n",
      "JQ340389.1 converted\n",
      "NC_005879.1 converted\n",
      "NC_003907.2 converted\n",
      "NC_024369.2 converted\n",
      "NC_025822.1 converted\n",
      "KF800937.1 converted\n",
      "NC_022747.1 converted\n",
      "NC_019518.1 converted\n",
      "NC_019457.1 converted\n",
      "KT160311.1 converted\n",
      "NC_027118.1 converted\n",
      "NC_026610.1 converted\n",
      "NC_003313.1 converted\n",
      "NC_023569.1 converted\n",
      "KJ936628.1 converted\n",
      "NC_023863.2 converted\n",
      "KJ502657.1 converted\n",
      "NC_023605.1 converted\n",
      "NC_023568.1 converted\n",
      "NC_010275.1 converted\n",
      "NC_004456.1 converted\n",
      "JF713456.1 converted\n",
      "HQ641341.1 converted\n",
      "NC_027368.1 converted\n",
      "KP972569.1 converted\n",
      "NC_004306.1 converted\n",
      "NC_001956.1 converted\n",
      "NC_024791.1 converted\n",
      "KM224878.1 converted\n",
      "KJ619459.1 converted\n",
      "NC_021776.1 converted\n",
      "JQ780163.1 converted\n",
      "NC_021562.1 converted\n",
      "NC_021540.1 converted\n",
      "KC438283.1 converted\n",
      "NC_012662.1 converted\n",
      "NC_016162.1 converted\n",
      "NC_015209.1 converted\n",
      "NC_012757.1 converted\n",
      "NC_009016.1 converted\n",
      "NC_006294.1 converted\n",
      "NC_005949.1 converted\n",
      "NC_005948.1 converted\n",
      "NC_004736.1 converted\n",
      "NC_003327.2 converted\n",
      "NC_002363.1 converted\n",
      "NC_002362.1 converted\n",
      "HQ641354.1 converted\n",
      "HQ641353.1 converted\n",
      "HQ641352.1 converted\n",
      "HQ641351.1 converted\n",
      "HQ641350.1 converted\n",
      "HQ641349.1 converted\n",
      "HQ641348.1 converted\n",
      "HQ641346.1 converted\n",
      "HQ641344.1 converted\n",
      "HQ641343.1 converted\n",
      "HG803181.1 converted\n",
      "AB572858.1 converted\n",
      "FN297812.1 converted\n",
      "AB043679.1 converted\n",
      "AB043678.1 converted\n",
      "AB002632.1 converted\n",
      "AB012574.1 converted\n",
      "AB012573.1 converted\n",
      "AB374228.1 converted\n",
      "D89074.1 converted\n",
      "KC131130.1 converted\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "from Bio import SeqIO\n",
    "import re\n",
    "\n",
    "keyword=\"(Vibrio phage|vibriophage).*genome\"\n",
    "accs=[]\n",
    "titles=[]\n",
    "acdict={}\n",
    "\n",
    "splist=open(\"/Users/jmb/Downloads/nuccore_result-1.txt\").readlines()\n",
    "for i in range(1,len(splist),4):\n",
    "    if re.search(keyword, splist[i]):\n",
    "        name=splist[i].split(\",\")[0].split(\".\")[1]\n",
    "        name2=re.sub(r\"complete genome\\n|genomic sequence\\n\",\"\", name)\n",
    "        if name not in titles:\n",
    "            titles.append(name2)\n",
    "            acc=splist[i+2].split(\" \")[0]\n",
    "            accs.append(acc)\n",
    "            acdict[acc]=name2\n",
    "accs_str=\",\".join(accs)\n",
    "\n",
    "import os\n",
    "\n",
    "#os.mkdir(\"./VibGenomes\")\n",
    "Entrez.email=\"julia.brown@einstein.yu.edu\"\n",
    "for a in accs:\n",
    "    handle=Entrez.efetch(db=\"nucleotide\", id=str(a), rettype=\"gb\", retmode=\"text\")\n",
    "    filename=\"./VibGenomes/\"+a+\".gbk\"\n",
    "    out=open(filename, \"w\")\n",
    "    out.write(handle.read())\n",
    "    out.close()\n",
    "    handle.close()\n",
    "\n",
    "for a in accs:\n",
    "    SeqIO.convert(\"./VibGenomes/\"+a+\".gbk\",\"gb\",\"./VibGenomes/\"+a+\".fasta\",\"fasta\")\n",
    "    print(a+\" converted\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "from Bio.SeqUtils import GC\n",
    "\n",
    "taxdict={}\n",
    "sizedict={}\n",
    "out=open(\"vib.genomeinfo.txt\",\"w\")\n",
    "\n",
    "for phage in accs:\n",
    "    gbk=SeqIO.parse(\"./VibGenomes/\"+phage+\".gbk\", \"genbank\")\n",
    "    for rec in gbk:\n",
    "        taxonomy=rec.annotations[\"taxonomy\"]\n",
    "        if len(taxonomy)>4:\n",
    "            tax=taxonomy[3]\n",
    "        else:\n",
    "            tax=taxonomy[-1]\n",
    "        taxdict[phage]=tax\n",
    "        sizedict[phage]=len(rec.seq)\n",
    "        gc_content=GC(rec.seq)\n",
    "        out.write(phage+\"\\t\"+acdict[phage]+\"\\t\"+tax+\"\\t\"+str(len(rec.seq))+\"\\t\"+str(gc_content)+\"\\n\")\n",
    "out.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_019713.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_019713.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KJ572844.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KJ572844.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KP280062.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KP280062.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_019722.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_019722.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KM612265.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KM612265.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KM612263.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KM612263.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KM612262.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KM612262.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KM612260.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KM612260.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_027393.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_027393.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KM612261.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KM612261.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_015159.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_015159.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_027397.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_027397.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_013651.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_013651.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KP280063.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KP280063.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_015157.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_015157.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_015158.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_015158.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_016567.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_016567.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_005891.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_005891.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KP671755.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KP671755.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KJ545483.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KJ545483.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ317393.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ317393.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_005083.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_005083.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_021073.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_021073.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_020488.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_020488.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/JQ340389.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/JQ340389.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_005879.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_005879.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_003907.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_003907.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_024369.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_024369.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_025822.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_025822.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KF800937.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KF800937.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_022747.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_022747.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_019518.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_019518.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_019457.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_019457.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KT160311.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KT160311.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_027118.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_027118.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_026610.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_026610.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_003313.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_003313.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_023569.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_023569.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KJ936628.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KJ936628.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_023863.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_023863.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KJ502657.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KJ502657.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_023605.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_023605.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_023568.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_023568.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_010275.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_010275.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_004456.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_004456.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/JF713456.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/JF713456.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641341.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641341.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_027368.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_027368.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KP972569.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KP972569.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_004306.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_004306.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_001956.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_001956.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_024791.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_024791.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KM224878.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KM224878.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KJ619459.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KJ619459.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_021776.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_021776.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/JQ780163.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/JQ780163.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_021562.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_021562.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_021540.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_021540.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KC438283.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KC438283.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_012662.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_012662.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_016162.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_016162.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_015209.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_015209.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_012757.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_012757.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_009016.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_009016.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_006294.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_006294.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_005949.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_005949.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_005948.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_005948.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_004736.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_004736.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_003327.2.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_003327.2trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_002363.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_002363.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/NC_002362.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/NC_002362.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641354.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641354.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641353.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641353.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641352.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641352.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641351.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641351.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641350.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641350.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641349.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641349.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641348.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641348.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641346.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641346.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641344.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641344.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HQ641343.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HQ641343.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/HG803181.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/HG803181.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB572858.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB572858.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/FN297812.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/FN297812.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB043679.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB043679.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB043678.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB043678.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB002632.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB002632.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB012574.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB012574.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB012573.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB012573.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/AB374228.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/AB374228.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/D89074.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/D89074.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n",
      "\n",
      "tRNAscan-SE v.1.3.1 (January 2012) - scan sequences for transfer RNAs\n",
      "\n",
      "  Please cite: \n",
      "\tLowe, T.M. & Eddy, S.R. (1997) \"tRNAscan-SE: A program for\n",
      "\timproved detection of transfer RNA genes in genomic sequence\"\n",
      "\tNucl. Acids Res. 25: 955-964.\n",
      "\n",
      "  This program uses a modified, optimized version of tRNAscan v1.3\n",
      "  (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),\n",
      "  a new implementation of a multistep weight matrix algorithm\n",
      "  for identification of eukaryotic tRNA promoter regions\n",
      "  (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),\n",
      "  as well as the RNA covariance analysis package Cove v.2.4.2\n",
      "  (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).\n",
      "\n",
      "------------------------------------------------------------\n",
      "Sequence file(s) to search:  ./VibGenomes/KC131130.1.fasta\n",
      "Search Mode:                 General\n",
      "Results written to:          ./VibGenomes/vib_tRNA/KC131130.1trnas.txt\n",
      "Output format:               Tabular\n",
      "Searching with:              tRNAscan + EufindtRNA -> Cove\n",
      "Covariance model:            TRNA2.cm\n",
      "tRNAscan parameters:         Strict\n",
      "EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)\n",
      "Search log saved in:         \n",
      "\n",
      "Pseudogene checking disabled\n",
      "------------------------------------------------------------\n",
      "\n",
      "No tRNAs found.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#os.mkdir(\"./VibGenomes/vib_tRNA/\")\n",
    "out=open(\"vib_trna_scan.sh\",\"w\")\n",
    "for phage in accs:\n",
    "    out.write(\"tRNAscan-SE -o ./VibGenomes/vib_tRNA/\"+phage+\"trnas.txt -G -D ./VibGenomes/\"+phage+\".fasta\\n\")\n",
    "out.close()\n",
    "!bash vib_trna_scan.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "there are a total of 92 phages but only 11 have tRNAs\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "\n",
    "names=glob.glob(\"./VibGenomes/vib_tRNA/*\")\n",
    "\n",
    "tRNA_phages=[]\n",
    "\n",
    "for n in accs:\n",
    "    tfile=\"./VibGenomes/vib_tRNA/\"+n+\"trnas.txt\"\n",
    "    if os.path.getsize(tfile)>0:\n",
    "        #print(\"phage \"+phage+\" has tRNAs\")\n",
    "        tRNA_phages.append(n)\n",
    "    #else:\n",
    "     #   print(\"phage \"+phage+\" does not have tRNAs\")\n",
    "\n",
    "print(\"there are a total of \"+str(len(names))+\" phages but only \"+str(len(tRNA_phages))+\" have tRNAs\")\n",
    "\n",
    "out=open(\"vib_tRNA_summary.txt\",\"w\")\n",
    "out.write(\"phage_short\\tphage_full\\tbound1\\tbound2\\ttRNA\\tanti_codon\\tintron_bound1\\titron_bound2\\tscore\\n\")\n",
    "\n",
    "for phage in tRNA_phages:\n",
    "    info=open(\"./VibGenomes/vib_tRNA/\"+phage+\"trnas.txt\").readlines()\n",
    "    for line in info[3:]:\n",
    "        vec=line.split(\"\\t\")\n",
    "        \n",
    "        out.write(acdict[phage]+\"\\t\"+vec[0]+\"\\t\"+vec[2]+\"\\t\"+vec[3]+\"\\t\"+vec[4]+\"\\t\"+vec[5]+\"\\t\"+vec[6]+\"\\t\"+vec[7]+\"\\t\"+vec[8])\n",
    "out.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\n",
       "Attaching package: dplyr\n",
       "\n",
       "The following object is masked from package:stats:\n",
       "\n",
       "    filter\n",
       "\n",
       "The following objects are masked from package:base:\n",
       "\n",
       "    intersect, setdiff, setequal, union\n",
       "\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<StrVector - Python:0x10afec8c0 / R:0x108307698>\n",
       "[str, str, str, ..., str, str, str]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%load_ext rpy2.ipython\n",
    "%R library(dplyr);library(magrittr); library(ggplot2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n",
      "ID: KC131130.1\n",
      "Name: KC131130\n",
      "Description: Vibrio phage VP4B, partial genome.\n",
      "Number of features: 211\n",
      "/comment=##Assembly-Data-START##\n",
      "Assembly Method       :: SOAPdenovo v. 1.06\n",
      "Sequencing Technology :: Illumina\n",
      "##Assembly-Data-END##\n",
      "/sequence_version=1\n",
      "/source=Vibrio phage VP4B\n",
      "/taxonomy=['Viruses', 'dsDNA viruses, no RNA stage', 'Caudovirales', 'Myoviridae']\n",
      "/keywords=['']\n",
      "/references=[Reference(title='Complete genome sequence of a novel phiKZ-like Vibrio phage', ...), Reference(title='Direct Submission', ...)]\n",
      "/accessions=['KC131130']\n",
      "/data_file_division=PHG\n",
      "/date=26-DEC-2013\n",
      "/organism=Vibrio phage VP4B\n",
      "/gi=432142535\n",
      "Seq('GTCAACCAAAAAGACATCCTAAGTGGACACTCTTCGAAATGCACCAACGAATTC...ACG', IUPACAmbiguousDNA())\n"
     ]
    }
   ],
   "source": [
    "for phage in accs:\n",
    "    gbk=SeqIO.parse(\"./VibGenomes/\"+phage+\".gbk\", \"genbank\")\n",
    "    print(rec)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
